mite_selenium/__init__.py

```diff
@@ -186,10 +212,18 @@ def _wait_for(self, condition_func, locator, err_msg, timeout=5):
     def switch_to_iframe(self, locator):
         self._remote.switch_to.frame(self._remote.find_element(*locator))
 
+    def add_request_interceptor(self, request_interceptor):
+        if self.is_selenium_wire:
+            self._remote.request_interceptor = request_interceptor
```

Comment:
`else: raise Exception("request interceptor not supported")`

(but see above about using selenium-wire unconditionally, which makes this comment go away)